test/test-get-memory.c

```diff
@@ -40,5 +42,6 @@ TEST_IMPL(get_memory) {
 #else
   ASSERT(total_mem > free_mem);
 #endif
+  ASSERT(available_mem <= free_mem);
```

Comment:
```suggestion
  ASSERT_LE(available_mem, free_mem);
```
New code should use the new-style macros. I really should move everything ever to the new macros sometime but blergh, so tedious.